lib/browser_client.dart

```diff
@@ -59,13 +60,12 @@ class BrowserClient extends BaseClient {
 
       reader.onLoad.first.then((_) {
         var body = reader.result as Uint8List;
-        completer.complete(new StreamedResponse(
-            new ByteStream.fromBytes(body),
+        completer.complete(new Response(
+            xhr.responseUrl,
             xhr.status,
-            contentLength: body.length,
-            request: request,
-            headers: xhr.responseHeaders,
-            reasonPhrase: xhr.statusText));
+            reasonPhrase: xhr.statusText,
+            body: new ByteStream.fromBytes(body),
```

Comment:
This also shouldn't be a `ByteStream`. In this case, just `new Stream.fromIterable([body])` should work.